service/src/main/java/cz/muni/fi/pa165/soccermanager/service/MatchService.java

```diff
@@ -1,17 +1,24 @@
 package cz.muni.fi.pa165.soccermanager.service;
 
 import cz.muni.fi.pa165.soccermanager.entity.Match;
+import cz.muni.fi.pa165.soccermanager.entity.Team;
+import cz.muni.fi.pa165.soccermanager.enums.StadiumEnum;
 
+import java.time.LocalDate;
 import java.util.List;
 
 /**
- * @author 445720 Martin Hamernik
- * @version 11/16/2017.
+ * An interface that defines a service access to the Match entity
+ * @author 456519 Filip Lux
+ * @version 11/23/2017.
  */
 public interface MatchService {
-    Match fetchById(long matchId);
-    List<Match> fetchAll();
-    void insert(Match match);
+    Match findById(long matchId);
+    List<Match> findAll();
+    List<Match> findByDate(LocalDate date);
+    List<Match> findByTeam(Team team);
+    List<Match> findByStadium(StadiumEnum stadium);
+    void create(Match match);
```

Comment:
Create should return instance of Match inserted into db. Sorry I made a mistake in interaface design